### DISCUSSION TITLE
Does Wix4.0.1 support dual-purpose bundle?

### DISCUSSION THREAD

**Discussion by Federau** [wixtoolset/issues#7796] [Questions] [answered]
I noticed that in wix4 .,dual-purpose msi is supported by perUserOrMachine  attribute. but when I tried to make a bundle, and add dual-purpose package inside,  I got an error:
 error WIX1133: Bundles require a package to be either per-machine or per-user. The MSI 'XXXSetup' ALLUSERS Property is set to '2' which may change from per-user to per-machine at install time. 
so currently, does wix 4.0.1 support dual-purpose bundle? or I just use it in a wrong way.
Thanks in advance.

**Comment by robmen** [comment, marked as answer]
Nope. No one has implemented that feature in bundles, yet.

You must set the install scope for MSI packages at bundle build time.

**Reply by Federau** [reply]
Thanks, so Can we determine the install scope in a CustomBootstrapperApplication?  like using Engine to set Variable AllUsers maybe?

**Reply by robmen** [reply]
> Nope.

> You must set the install scope for MSI packages at bundle build time.

> No one has implemented that feature in bundles, yet.

**Reply by LosWolfos** [reply]
Still open.
https://github.com/wixtoolset/issues/issues/4914
